### PR TITLE
reduces calc overhead of suffix helper method

### PR DESCRIPTION
**Pull request by jtbg**
_if only there were some sort of tool we could use to measure the performance gains_

### CODE REVIEW

**Review by kbrock — APPROVED** [review]
this is nice.

could you squash your commits?
(disclaimer: I'm not a maintainer)

**Comment by kbrock** [comment]
Aah, I was confused because you commented on a commit that had changes not here.
This PR is just picking off the obvious win. 👍

**Comment by kbrock** [comment]
hmm... could you remove the merge commit, and rebase this onto master?

(sorry if you already know this stuff)

```
git fetch upstream
git checkout scale_helper_optimization
git branch scale_helper_optimization_bak # just in case
git reset --hard HEAD~1 # drop the merge commit
git rebase upstream/master

git diff scale_helper_optimization_bak # should be the same
git log -3 # this branch should just have a single commit
```

**Comment by jtbg** [comment]
WHat does `git branch scale_helper_optimization_bak` do here? I've never seen that before

**Comment by jtbg** [comment]
This is why I always get @emgreen33 to untangle my git spaghetti for me :rofl: 
Thanks for the help!

File: lib/benchmark/timing.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Benchmark
-  # Perform caclulations on Timing results.
```
Comment (nateberkopec) [inline lib/benchmark/timing.rb:2]: I did it for the lulz!

**Pull request merged**